### DISCUSSION TITLE
Next/image does not work with position attributes such as "absolute", "relative", etc.

### DISCUSSION THREAD

**Discussion by dancherb** [vercel/next.js#24101] [Help]
### What version of Next.js are you using?

10.0.7

### What version of Node.js are you using?

12.18.3

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

Local development

### Describe the Bug

The "position" key in the style object supplied to next's Image component has no effect.

### Expected Behavior

Next's Image component was expected to behave the same as the standard img component with regards to styles.

### To Reproduce

```
<NextImage width={100} height={100} src={image} style={{
                                position: "absolute",
                                top: 0,
                                left: 0
}}/>
```

**Comment by ronfogel** [comment]
Instead all your props that you currently have, use `layout="fill"`

**Comment by awareness481** [comment]
These are probably overwritten by `next/image`'s default inline styles. You could try using `!important`. Moreover `next/image` doesn't accept a `style` prop, you can try using `className` instead. https://nextjs.org/docs/api-reference/next/image#other-props

However, this isn't a bug.

**Comment by nersoh** [comment]
Duplicate of #23095

**Comment by MoOx** [comment]
Not a bug indeed, per documentation.
